### PR TITLE
Change OverlayBD to lowercase; Update README to make a better logic; Add Performance doc

### PR DESCRIPTION
**Pull request by beef9999**
Signed-off-by: Bob Chen <beef9999@qq.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,61 +5,63 @@ Accelerated Container Image is an open-source implementation of paper ["DADI: Bl
...
+
+## Components
+
+* [overlaybd](https://github.com/alibaba/overlaybd)
+
+    Overlaybd provides a merged view of block-based layer sequence as a third-party backing-store of tgt, which is an user space iSCSI target framework.
+
+* overlaybd-snapshotter
+
+    It's just what this repo focuses on: a [containerd](https://containerd.io/) snapshotter plugin for overlaybd image. This snapshotter is compatible for OCI image, as well as overlayfs snapshotter.
```
Comment (liulanzheng) [inline README.md:33]: this repo will have more components，so i think 'It's just what this repo focuses on:' is not suitable

File: README.md [hunk 2]
```diff
@@ -5,61 +5,63 @@ Accelerated Container Image is an open-source implementation of paper ["DADI: Bl
...
+
+    We also provide a modified CLI tool(ctr) to facilitate image download and custom conversion, between traditional OCI tarball format and overlaybd's obd format.
```
Comment (liulanzheng) [inline README.md:35]: I think 'image download' is ambiguous, it actually does not download the image.
'overlaybd's obd format', just like 'overlaybd format' or 'overlaybd-format' is ok.

Comment (beef9999) [reply README.md:35]: download -> pull

overlaybd's obd format  -> overlaybd format

File: README.md (hunk 1 above)
Comment (beef9999) [reply README.md:33]: It is a containerd snapshotter plugin for overlaybd image

**Review by liulanzheng — APPROVED** [review]
LGTM

**Pull request merged**